main/autogen/subclasses.cc

```diff
@@ -158,6 +158,7 @@ vector<string> Subclasses::genDescendantsMap(Subclasses::Map &childMap, vector<s
         auto descendants = Subclasses::descendantsOf(childMap, parentName);
         if (!descendants) {
             descendantsMap[parentName];
+            continue;
         }
 
         descendantsMap.emplace(parentName, *descendants);
```

Comment:
Two suggestions, both optional:

1. Add a comment explaining that the `descendantsMap[parentName];` statement is (I think???) there for its side effect of creating a value for the key `parentName` if it doesn't already exist. (I always find this behavior of `operator[]` confusing.)
2. Instead of `continue`, move the `emplace` into an `else`.